css/stanford_jumpstart_engineering.css

```diff
@@ -51,4 +51,10 @@
   .style-custom-styles-rich #block-bean-jse-linked-logo-block p a {
     padding-bottom: 10px;
   }
+}
+
+/* SOE footer logo for cardinal style */
+
+.views-grid-four .views-row.views-row-even:nth-of-type(4n) {
+    margin-right: 0;
 }
```

Comment:
@josephgknox: It looks like you're missing a carriage return at the end of the file.